gulpfile.babel.js

```diff
@@ -364,26 +377,30 @@ gulp.task('components', ['demos'], () => {
       });
     })())
     .pipe(applyTemplate())
-    .pipe($.rename(path => path.basename = 'index'))
+    .pipe($.rename(path => {
+      path.basename = 'index';
+    }))
```

Comment:
To avoid the fat arrow implicit return, which in this case triggers a [no-return-assign](http://eslint.org/docs/rules/no-return-assign) error. This way the function returns nothing, as intended.